modules/mono/editor/bindings_generator.cpp

```diff
@@ -2662,6 +2670,10 @@ Error BindingsGenerator::_generate_cs_method(const BindingsGenerator::TypeInterf
 						"' from the editor API. Core API cannot have dependencies on the editor API.");
 	}
 
+	if (p_imethod.is_virtual && p_use_span) {
+		return OK;
+	}
+
 	String method_bind_field = CS_STATIC_FIELD_METHOD_BIND_PREFIX + itos(p_method_bind_count);
```

Comment:
We should avoid generating more than one method bind field. We can assume if `p_use_span == true` we don't need to generate it because the array overload will have generated it for us.

And since `p_method_bind_count` is increased at the end of this method, make sure to use `p_method_bind_count - 1` here when `p_use_span == true` so it references the right field.

Reply:
Done, the wrapper code looks like the following now:
<details>

```csharp
[DebuggerBrowsable(DebuggerBrowsableState.Never)]
private static readonly IntPtr MethodBind51 = ClassDB_get_method_with_compatibility(NativeName, MethodName.SetEmissionRectExtents, 743155724ul);

[EditorBrowsable(EditorBrowsableState.Never)]
public unsafe void SetEmissionRectExtents(Vector2 extents)
{
    NativeCalls.godot_icall_1_34(MethodBind51, GodotObject.GetPtr(this), &extents);
}

[DebuggerBrowsable(DebuggerBrowsableState.Never)]
private static readonly IntPtr MethodBind52 = ClassDB_get_method_with_compatibility(NativeName, MethodName.GetEmissionRectExtents, 3341600327ul);

[EditorBrowsable(EditorBrowsableState.Never)]
public Vector2 GetEmissionRectExtents()
{
    return NativeCalls.godot_icall_0_35(MethodBind52, GodotObject.GetPtr(this));
}

[DebuggerBrowsable(DebuggerBrowsableState.Never)]
private static readonly IntPtr MethodBind53 = ClassDB_get_method_with_compatibility(NativeName, MethodName.SetEmissionPoints, 1509147220ul);

[EditorBrowsable(EditorBrowsableState.Never)]
public void SetEmissionPoints(Vector2[] array)
{
    NativeCalls.godot_icall_1_204(MethodBind53, GodotObject.GetPtr(this), array);
}

[EditorBrowsable(EditorBrowsableState.Never)]
public void SetEmissionPoints(ReadOnlySpan<Vector2> array)
{
    NativeCalls.godot_icall_1_204(MethodBind53, GodotObject.GetPtr(this), array);
}

[DebuggerBrowsable(DebuggerBrowsableState.Never)]
private static readonly IntPtr MethodBind54 = ClassDB_get_method_with_compatibility(NativeName, MethodName.GetEmissionPoints, 2961356807ul);

[EditorBrowsable(EditorBrowsableState.Never)]
public Vector2[] GetEmissionPoints()
{
    return NativeCalls.godot_icall_0_205(MethodBind54, GodotObject.GetPtr(this));
}

[DebuggerBrowsable(DebuggerBrowsableState.Never)]
private static readonly IntPtr MethodBind55 = ClassDB_get_method_with_compatibility(NativeName, MethodName.SetEmissionNormals, 1509147220ul);

[EditorBrowsable(EditorBrowsableState.Never)]
public void SetEmissionNormals(Vector2[] array)
{
    NativeCalls.godot_icall_1_204(MethodBind55, GodotObject.GetPtr(this), array);
}

[EditorBrowsable(EditorBrowsableState.Never)]
public void SetEmissionNormals(ReadOnlySpan<Vector2> array)
{
    NativeCalls.godot_icall_1_204(MethodBind55, GodotObject.GetPtr(this), array);
}
```

</details>